study.md

```diff
@@ -21,18 +21,53 @@ There is no requirement to follow any links from these pages.
 After doing all the required readings please tell us anything you were confused about.
 
 ```md
-<!-- your answer here -->
+***********************************
+in this code :
+
+function counter() {
+  for (var count = 1; ; count++) {  // infinite loop
+    console.log(count + "A"); // until 5
+      if (count === 5) {
+        return;
+      }
+      console.log(count + "B");  // until 4
+    }
+  console.log(count + "C");  // never appears
+}
+
+counter();
+
+when do we use a break statement and when do we use a return statement?
```

Comment:
When ever you want the function to break the loop. Hopefully today's material helped with this.